tuxemon/config.py

```diff
@@ -107,6 +107,15 @@ def __init__(self, config_path: Optional[str] = None) -> None:
         self.player_walkrate = cfg.getfloat("player", "player_walkrate")  # tiles/second
         self.player_runrate = cfg.getfloat("player", "player_runrate")  # tiles/second
 
+        # [controls]
```

Comment:
This is duplicated by existing code in config.py
```python
        # input config (None means use default for the platform)
        self.gamepad_deadzone = 0.25
        self.gamepad_button_map = None
        self.keyboard_button_map = get_custom_pygame_keyboard_controls(cfg)
```

Reply:
I made a very crude solution to this issue.. Let me know if this looks a little messy after I send the commit.